### ISSUE TITLE
chunk error upon restarting from opmd/adios2

### ISSUE THREAD

**Issue by wmles** [ComputationalRadiationPhysics/picongpu#4182]
when restarting from checkpoint on juwels [i.e. I edited the `submit.start` by hand and increased steps and appended `--checkpoint.restart`, so definitely the same executable] I get the following stderr (consistently, in several simulations):

```
The following modules were not unloaded:
  (Use "module --force purge" to unload all):

  1) Stages/2022
ln: failed to create symbolic link 'output': File exists
Unhandled exception of type 'St13runtime_error' with message 'Chunk does not reside inside dataset (Dimension on index 2. DS: 3449389056 - Chunk: 18446744071965943808)', terminating
Unhandled exception of type 'St13runtime_error' with message 'Chunk does not reside inside dataset (Dimension on index 2. DS: 3449389056 - Chunk: 18446744072469571584)', terminating
[...several hundreds more of those chunk-error-lines...]
```

As Rene noticed, the chunk index is very large, it's something like FFFFFFFF9812A800 which looks suspiciously like a small negative number (?). 
As Franz sais, the message is written by openPMD, (inside `RecordComponent::loadChunk<>()`), but the error is apparently happening earlier, inside PIConGPU's index calculation.

posting this here upon Klauss request, so the explanation/solution will be searchable

**Comment by sbastrakov** [comment]
~~Just to make sure, this happens with a relatively fresh PIConGPU `dev` version? E.g. we had a fix https://github.com/ComputationalRadiationPhysics/picongpu/pull/4037 several month ago that may have caused a similar behavior~~

Edit: that linked PR fixed a bug that was happening with moving window enabled, which is not the case here. So the reason must be something else.

**Comment by steindev** [comment]
@wmles does the issue still exist?

**Comment by PrometheusPi** [comment]
ping @wmles

**Comment by PrometheusPi** [comment]
This does still exist - https://github.com/ComputationalRadiationPhysics/picongpu/issues/4570

**Comment by BrianMarre** [comment]
@chillenzer this issue has been fixed with #4573, please close this issue

**Comment by chillenzer** [comment]
Closed by trusting @BrianMarre that this is really fixed.
